ui/apps/platform/eslint-plugins/pluginLimited.js

```diff
@@ -0,0 +1,70 @@
+/* globals module require */
+
+const path = require('node:path');
+
+// Limited rules have exceptions via ignores property.
+// When ESLint plugin for Visual Studio Code has support for suppressions, they might supersede limited rules.
+
+const rules = {
+    // ESLint naming convention for positive rules:
+    // If your rule is enforcing the inclusion of something, use a short name without a special prefix.
+
+    'export-default-react': {
```

Comment:
I believe right now we still use explicit React imports (which is an older pattern), and this rule should work. But, since we're on React 18, we could potentially run into issues if we use the more modern pattern (no React imports). It may not be a big issue if we all stick with the current pattern and make sure any new developers also stick with it. Not sure there's a need to use the new patterns anyway.

Reply:
Also, I know this rule applies to React Components for now. Would we want it to expand to hooks? I know some hook files (e.g., useSnoozeCVEModal) don't have an explicit React import, so something like this may miss that. Something to consider.

Reply:
I changed the cues:
* `'.jsx'` or `'.tsx'` file name extension for React component
* `name.startsWith('use')` for hook (now that I think, I might need to require capital letter following to prevent unintended match with `'userWhatchamacallit`` for example.